test/helpers/kubectl.go

```diff
@@ -412,6 +412,57 @@ func (kub *Kubectl) BackgroundReport(commands ...string) (context.CancelFunc, er
 	return cancel, nil
 }
 
+// PprofReport runs pprof on cilium nodes each 5 minutes and saves the data
+// into the test-output.log
+func (kub *Kubectl) PprofReport() {
```

Comment:
Can you create an issue which tracks doing this for the runtime test suite as well?